bin/commands/generate.mjs

```diff
@@ -121,6 +131,11 @@ export default {
   async action(opts) {
     const docs = await loadAndParse(opts.input, opts.ignore);
     const releases = await parseChangelog(opts.changelog);
+
+    const typeMap = opts.typeMap
```

Comment:
Make opts.typeMap have a default path that goes to a fallback one we store in this repo. Better that the fallback also uses the schema we defined.